service.watchdog/service.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014 Thomas Amland
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+if __name__ == "__main__":
+    import os
+    import sys
+    import xbmcaddon
+    libs = os.path.join(xbmcaddon.Addon().getAddonInfo('path'), 'lib')
+    sys.path.append(libs)
```

Comment:
using `sys.path.append` is really a bad practice. You should be able to just import the module you need right?

```python
from lib.watchdog.core import main
main.main()
```

Reply:
Hmmm, without this how would I access let's say the watchdog package from the core package?

Reply:
if the watchdog is a script.module you just `import watchdog`